apple/RNCWebViewManager.m

```diff
@@ -104,6 +104,7 @@ - (RCTUIView *)view
 #endif
 
 #if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 150000 /* iOS 15 */
+RCT_EXPORT_VIEW_PROPERTY(textInteractionEnabled, BOOL)
```

Comment:
This is still gating it under iOS 15...?

Reply:
Created an iOS 14.5 section for this